hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -82,40 +150,41 @@
         }
       ]
     },
-    { name: "ES_CTRL",
-      desc: "ENTROPY_SRC control register",
-      swaccess: "rw",
-      hwaccess: "hro",
-      regwen:   "ES_REGEN",
-      fields: [
-        { bits: "0",
-          name: "INIT_ENT_SRC",
-          desc: "Setting this bit will initialize the entropy source function."
-        }
-      ]
-    },
     { name:     "ES_STATUS",
       desc:     "ENTROPY_SRC status register",
       swaccess: "ro",
-      hwaccess: "hrw",
+      hwaccess: "hwo",
       hwext: "true",
       fields: [
-        { bits: "0",
-          name: "ENTROPY_RDY",
-          desc: '''This is set when the entropy source FIFO has met or exceeded the ES_THRESH value. This status bit will
-                automatically clear once the FIFO depth drops below the threshold value. This bit will not set unless the
-                block is enabled.
-                '''
+        { bits: "2:0",
+          name: "FIFO0_DEPTH",
```

Comment:
i feel like if the field names here reflected the function (like the description), it would be clearer.

Also, assuming we resolve the issue above, and we can say the analog source uses the same clock as the lfsr, could we combine the fifo0/fifo1? It seems like these are mutually exclusive options.

Reply:
Odd - the bit "ENTROPY_RDY" is gone from my source. It has been deprecated. 
True for the next comment, if it is true. Need specification/information to prove.

Reply:
sounds good. See comment above, we can setup another meeting to discuss.